docs/transaction.rst

```diff
@@ -0,0 +1,57 @@
+Transaction Operations
+======================
+
+Transact operations are supported using context managers. The DynamoDB API has limits on the number of items in
+each request, but PynamoDB doesn't currently handle grouping or paginating, so this is something you must handle on your
+own.
+
+.. note::
+
+    An ordered array of up to 25 `TransactWriteItem` objects, each of which contains a `ConditionCheck`, `Put`, `Update`, or `Delete` object. These can operate on items in different tables, but the tables must reside in the same AWS account and Region, and no two of them can operate on the same item.
```

Comment:
Is this documenting the underlying DynamoDB API or does this talk in PynamoDB terms?

Reply:
this the underlying dynamodb api

Reply:
I'd keep our docs to be "pynamodb first". We only need to go into DynamoDB API (which is documented by AWS already) if we need to explain something that's unclear from the PynamoDB API.

Reply:
Perhaps something like "Keep in mind that DynamoDB imposes limits on the number of items that a single transaction can contain." with a link to their docs